python/pyspark/sql/pandas/functions.py

```diff
@@ -33,12 +37,8 @@ class PandasUDFType(object):
 
     GROUPED_MAP = PythonEvalType.SQL_GROUPED_MAP_PANDAS_UDF
 
-    COGROUPED_MAP = PythonEvalType.SQL_COGROUPED_MAP_PANDAS_UDF
-
     GROUPED_AGG = PythonEvalType.SQL_GROUPED_AGG_PANDAS_UDF
 
-    MAP_ITER = PythonEvalType.SQL_MAP_PANDAS_ITER_UDF
```

Comment:
My logic here is that, we keep `PythonEvalType.SQL_COGROUPED_MAP_PANDAS_UDF` and `PythonEvalType.SQL_MAP_PANDAS_ITER_UDF` as internal flags for `mapInPandas` or `applyInPandas`; however, users don't set it directly since only `PandasUDFType` is the API